src/apps/dashboard/routes/users/parentalcontrol.tsx

```diff
@@ -19,9 +20,12 @@ import Page from '../../../../components/Page';
 import prompt from '../../../../components/prompt/prompt';
 import ServerConnections from 'components/ServerConnections';
 
-type UnratedItem = {
+type ItemArr = {
```

Comment:
`Arr` is a misleading name since this isn't an array itself... maybe something like

```suggestion
type NamedItem = {
```